src/openrct2/object/ObjectTypes.h

```diff
@@ -41,12 +41,13 @@ enum class ObjectType : uint8_t
     audio,
     peepNames,
     peepAnimations,
+    climate,
```

Comment:
The function to remove unused objects also needs updating in order not to remove the climate object.